### PR TITLE
small fix

### PR DESCRIPTION
**Pull request by FrederikRothe**
## Describe your changes
file name fix, caused encoding errors.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [x] Have I requested and notified the boys for review of the pull-request?

### CODE REVIEW

File: modu/main.html
Comment (SadCheeto) [inline modu/main.html:1]: ???

File: metrics/test/data/test_data_file_length/data_file_length_invalid_symbols 2.cs [hunk 1]
```diff
@@ -243,7 +243,7 @@
 Console.WriteLine("Hello World!");
 Console.WriteLine("Hello World!");
 Console.WriteLine("Hello World!");
-Console.WriteLine("Hello World!");
+Console.WriteLine("Hello World!");f
```
Comment (SadCheeto) [inline metrics/test/data/test_data_file_length/data_file_length_invalid_symbols 2.cs:246]: ?

**Review by SadCheeto — COMMENTED** [review]
What is the point of this change - I am unsure?

File: modu/main.html
Comment (FrederikRothe) [reply modu/main.html:1]: problems with cpd and similar code cli commands 👍🏼

Comment (toffernator) [reply modu/main.html:1]: When running cpd on files with spaces in it, it crashed.

Comment (toffernator) [inline modu/main.html:1]: This file should be gitignored, not sure how it was accidentally committed, but it shouldn't be merged into main.

**Review by toffernator — APPROVED** [review]
I am approving it assuming you make sure `main.html` doesn't get merged into main.

**Pull request merged**